sdk/logs/src/main/java/io/opentelemetry/sdk/logs/export/BatchLogRecordProcessor.java

```diff
@@ -197,6 +199,12 @@ private Worker(
                   "The number of logs processed by the BatchLogRecordProcessor. "
                       + "[dropped=true if they were dropped due to high throughput]")
               .build();
+      logsExportFailureCounter =
+          meter
+              .counterBuilder("logsExportFailure")
```

Comment:
While this seems like a small change, I'm reluctant to make it because there have been some attempts to standardize the SDKs' internal telemetry (e.g. [OTEP#238](https://github.com/open-telemetry/oteps/pull/238)). 

The problem with continuing the pattern of these current metrics is that the structure doesn't conform to our [semantic convention recommendations](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/general/metrics.md).

- The unit is wrong - should probably be `{export}` instead of `1`
- The metric name doesn't include a namespace
- The attributes don't have a namespace

Extending the instrumentation extends bad patterns. Fixing the bad patterns exposes our users to breaking changes, only to have more later if / when semantic conventions emerge. So we appear to be stuck.

Reply:
@jack-berg agree with you here that current pattern (existing as well as any proposed metric in future) doesn't conform to semantic recommendations like metric name having namespace, well defined units, etc.
So we are stuck between extending new instrumentations/ rectifying' the existing instrumentations with bad semantics AND the recommended ones. Do let us know how the discussions go with this. As this will be applicable in general, not just here.